VimeoUpload/Upload/Operations/Async/ExportOperation.swift

```diff
@@ -167,14 +167,14 @@ public class ExportOperation: ConcurrentOperation
         // So I'm using this while loop to update a dynamic property instead, and KVO'ing on that [AH] 10/22/2015
         
         DispatchQueue.global(qos: .utility).async { [weak self] () -> Void in
-            while self?.exportSession.status == AVAssetExportSessionStatus.waiting || self?.exportSession.status == AVAssetExportSessionStatus.exporting
+            while self?.exportSession.status == .waiting || self?.exportSession.status == .exporting
```

Comment:
Hmm... I would prefer strongly recapturing `self` here before using it. If I remember correctly from Mike's talk a while ago, the reason is because the first time `self?` is used, the runtime will try to capture it, but for the second time, there's no guarantee that `self?` will exist.

Reply:
I'd rather not touch this to be honest unless we have identified an issue, which should then go on a separate ticket & PR. Off the top of my head if we capture `self` strongly there is a chance the while loop inside this closure may run forever. Imagine a scenario where the status of the `exportSession` remains set to `waiting` for an indefinite period, the closure would never finish and `self` would remain retained inside it.